src/lib/circuit_value.ts

```diff
@@ -43,6 +43,9 @@ type NonMethodKeys<T> = {
 }[keyof T];
 type NonMethods<T> = Pick<T, NonMethodKeys<T>>;
 
+/**
+ * @deprecated `CircuitValue` is deprecated in favor of `Struct`, which features a simpler API and better typing.
```

Comment:
nit: Should you link to `Struct` in the comment?

Reply:
How should we link? the `[[ Struct ]]` thing wasn't working